mentoring-wg/gsoc-org-admin-guide.md

```diff
@@ -0,0 +1,163 @@
+# CNCF GSoC admin guide
+
+## Introduction
+
+> **Note**
+> This is a guide for Google Summer of Code admins at CNCF.
+
+While Google [defines](https://developers.google.com/open-source/gsoc/help/responsibilities#org_admin_responsibilities) responsibilities for organization admins and gives some [tips](https://developers.google.com/open-source/gsoc/help/oa-tips), there are variations in how organizations run the program suitable for their communities.
+
+## Outline of responsibilities
+
+First of all, GSoC admins are required to read these documents from Google:
+
+* [GSoC admin responsibilities](https://developers.google.com/open-source/gsoc/help/responsibilities#org_admin_responsibilities)
+* [GSoC admin tips](https://developers.google.com/open-source/gsoc/help/oa-tips)
+
+In summary, responsibilities can be outlined as following.
+
+* Recruitment: GSoC admins are responsible for recruiting mentors from the CNCF communities and also contributors to participate in the program. They reach out to communities, promote the program, and answer questions from interested parties.
+* Mentor management: GSoC admins ensure that mentors are qualified and have the necessary experience to mentor contributors effectively. They provide guidance and feedback as needed to help mentors improve their mentoring skills.
+* Contributor management: GSoC admins set up the foundation for the communication between the mentors and the contributors. They provide guidance for contributors and mentors to connect, monitor the progress of projects, and provide non-technical support as needed.
+* Program coordination: GSoC admins coordinate the program logistics. This may include applying to the program, ranking proposals, setting deadlines, communicating with involved parties, and ensuring that all contributors and mentors have the resources they need to be successful.
+* Program evaluation: Admins evaluate the success of the program and identify areas for improvement. They do this by analyzing program data, collecting feedback from contributors and mentors, and making recommendations for future iterations of the program.
+
+Following are not responsibilities of a GSoC admin:
+* Verifying the eligibilities of contributors in terms of age, country of residence and similar criteria. These are done by Google.
+* Providing technical support to mentors or contributors about the project.
+* Technically or not-techically mentoring the contributors, other than giving general guidance and replying questions by pointing at possible places to find an answer.
```

Comment:
Can we add an example of what they shouldn't be doing? "Technically or not-technically" might not be so obvious to folks reading.

Reply:
Very good point.

I've committed your improvement suggestion about being more specific with "technical mentoring". I think we're fine there.

About non-technical mentoring for contributors: I wanted to mean, for example, admins don't have to give career advice OR don't have to suggest what skills to gain.

Let me write something for that aspect, if that makes sense.

Reply:
I added this sentence:

> Providing non-technical mentoring to the contributors (e.g. career advice, personal advice, etc.).